gc_functions/db_scripts/update_scryfall_bulk/index.js

```diff
@@ -1,34 +1,36 @@
-const argv = require('yargs').argv;
-const saveScryfallBulk = require('./saveScryfallBulk');
-const mongoBulkImport = require('./mongoBulkImport');
-const getLanguageCards = require('./getLanguageCards');
-const SOURCE_JSON_URI = './bulk_data/bulk_all_cards.json';
+const argv = require("yargs").argv;
+const saveScryfallBulk = require("./saveScryfallBulk");
+const mongoBulkImport = require("./mongoBulkImport");
+const getLanguageCards = require("./getLanguageCards");
+const SOURCE_JSON_URI = `${__dirname}/bulk_data/bulk_all_cards.json`;
```

Comment:
The script's new execution location (from `package.json`) differed from the location of `index.js` (where we normally ran it from), so this file was unreachable without an absolute path reference